runtime/testruntime/testruntime.go

```diff
@@ -224,11 +228,22 @@ func NewInstanceForProject(t TestingT, name string) (*runtime.Runtime, string) {
 	return rt, inst.ID
 }
 
-func NewInstanceForDruidProject(t TestingT) (*runtime.Runtime, string) {
+func NewInstanceForDruidProject(t *testing.T) (*runtime.Runtime, string, error) {
+	_, currentFile, _, _ := goruntime.Caller(0)
+	envPath := filepath.Join(currentFile, "..", "..", ".env")
```

Comment:
Won't this resolve to `<repo>/runtime/.env`, not `<repo>/.env`? Can you test any changes to confirm?

Reply:
Fixed.